### ISSUE TITLE
Can't get GitLab PR decoration to work on 8.1

### ISSUE THREAD

**Issue by teake** [mc1arke/sonarqube-community-branch-plugin#109]
Hi, first of all thanks for the great plugin! 

**Describe the bug**
I'm trying to get PR decoration to work on GitLab, but to no avail.  If somebody can point to where I'm doing it wrong, I'd  be much obliged!

**To Reproduce**
1. I've compiled from the sq-8_1-support branch
2. Follow instructions for GitLab here: https://docs.sonarqube.org/latest/analysis/pr-decoration/
3. Follow instructions for SonarScanner CLI here: https://docs.sonarqube.org/latest/analysis/gitlab-cicd/

**Expected behavior**
That the MR is decorated like in https://gitlab.com/tisoft/sonarqube-community-branch-plugin-gitlab-test/merge_requests/1.

**Software Versions**
 - GitLab 12.8.1
 - SonarQube Version: 8.1.0.31237
 - Plugin Version: build from sq-8_1-support at commit 87fc1b537a223c38e6297369a10d731587c24749

**Comment by mc1arke** [comment]
Have you checked the logs from the CE components of Sonarqube; typically `$SONARQUBE_HOME/logs/ce.log`? It may also be useful to turn on debug logging for CE by adding the line `sonar.log.level.ce=INFO` into `SONARQUBE_HOME/conf/sonar.properties`.

It's going to be difficult to diagnose your issue without some idea of where Sonarqube is actually getting to in the decoration process.

**Comment by teake** [comment]
Thanks for the quick reply!

I'm actually not sure what I should be looking for in the logs. Doing a `grep -ri 'deco' <sonarhome>/log` yields

```
./ce.log:2020.02.26 15:11:25 DEBUG ce[AXCCDXAO3VL0TdkCDRrU][c.g.m.s.p.c.p.PullRequestPostAnalysisTask] found 3 pull request decorators
./ce.log:2020.02.26 15:11:25 INFO  ce[AXCCDXAO3VL0TdkCDRrU][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=SUCCESS | time=1ms
./ce.log:2020.02.26 15:14:42 DEBUG ce[AXCCEHN43VL0TdkCDRrW][c.g.m.s.p.c.p.PullRequestPostAnalysisTask] found 3 pull request decorators
./ce.log:2020.02.26 15:14:42 INFO  ce[AXCCEHN43VL0TdkCDRrW][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=SUCCESS | time=1ms
./ce.log:2020.02.26 15:25:52 DEBUG ce[AXCCGquk3VL0TdkCDRrY][c.g.m.s.p.c.p.PullRequestPostAnalysisTask] found 3 pull request decorators
./ce.log:2020.02.26 15:25:52 INFO  ce[AXCCGquk3VL0TdkCDRrY][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=SUCCESS | time=2ms
```
I cannot find a mention of the info lines written in [`GitlabServerPullRequestDecorator#decorateQualityGateStatus`](https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/sq-8_1-support/src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabServerPullRequestDecorator.java#L98) (e.g. _"starting to analyze with "_ is not present in the logs). 

Also, it might be worth mentioning that the default branch and target of the MR is not `master`.
But the PR Quality Gate analysis does show up correctly in SonarQube.

**Comment by teake** [comment]
Ok, I cleared the logs and looked again. After spotting

```
2020.02.27 07:41:09 DEBUG ce[AXCFl5Sc3VL0TdkCDRrl][c.g.m.s.p.c.p.PullRequestPostAnalysisTask] No ALM has been set on the current project
```
in `ce.log` I realized I still had to set the Pull Request Decoration in Project -> Administration -> General -> Pull Requestion Decoration. This is mentioned in https://docs.sonarqube.org/latest/analysis/pr-decoration/ but I managed to overlook that :(.

That got me a little further but then I hit a 404 because the PR id was set wrongly. After following https://community.sonarsource.com/t/gitlab-pull-request-decoration-is-not-working/17974 and removing the parameters to the scanner, it worked!

So again thanks for the great plugin, and sorry for wasting your time.
